README.md

```diff
@@ -11,10 +11,12 @@ The name fortio comes from greek φορτίο which is load/burden.
 2. `go get -u istio.io/fortio`
 3. you can now run `fortio` (from your gopath bin/ directory)
 
-Or
-
-`docker run istio/fortio arguments...`
+Or use docker, for instance:
 
+```shell
+docker run -p 8080:8080 -p 8079:8079 istio/fortio server &
+docker run istio/fortio load http://$HOSTNAME:8080/
+```
 
 ## Command line arguments
 
```

Comment:
I would split command line arguments into 3 bullets, for clarity.

Reply:
not sure I understand this point, but I added bullets for the 3 types of example if that's what you meant - lmk